docs_src/microservices/application/Triggers.md

```diff
@@ -1,109 +1,198 @@
-Triggers determine how the app functions pipeline begins execution. The trigger is determined by the `configuration.toml` file located in the `/res` directory under a section called `[Binding]`. Check out the [Configuration Section](../GeneralAppServiceConfig/) for more information about the toml file.
+# Application Service Triggers
 
-## EdgeX Message Bus Trigger
+### Introduction
 
-An EdgeX Message Bus trigger will execute the pipeline every time data is received from the configured Edgex Message Bus `SubscribeTopic`.  The EdgeX Message Bus is the central message bus internal to EdgeX and has a specific message envelope that wraps all data published to this message bus.
+Triggers determine how the App Functions Pipeline begins execution. The trigger is determined by the `[Trigger]` configuration section in the  `configuration.toml` file.   
 
-There currently are three implementations of the EdgeX Message Bus available to be used. These are `ZeroMQ`, `MQTT` & `Redis Streams`. The implementation type is selected via the `[MessageBus]` configuration described below.
+!!! edgey "Edgex 2.0"
+    For Edgex 2.0 the `[Binding]` configuration section has been renamed to `[Trigger]`. The  `[MessageBus]` section has been renamed to `EdgexMessageBus` and moved under the `[Trigger]` section. The `[MqttBroker]` section has been renamed to `ExternalMqtt` and moved under the `[Trigger]` section.
+
+There are 4 types of `Triggers` supported in the App Functions SDK which are discussed in this document
+
+1. **[EdgeX Message Bus](#edgex-messagebus-trigger)** - Default Trigger for most use cases as this is how the App Services receive Events from EdgeX Core Data and/or Devices Services
+2. **[External MQTT](#external-mqtt-trigger)** - Useful when receiving commands from a Cloud MQTT broker.
+3. **[HTTP](#http-trigger)** - Useful during development and testing of custom functions. 
+4. **[Custom](#custom-triggers)** - Allows custom Application Services to implement their own Custom Trigger
+
+## EdgeX MessageBus Trigger
+
+An EdgeX MessageBus trigger will execute the pipeline every time data is received from the configured Edgex MessageBus `SubscribeTopics`.  The EdgeX MessageBus is the central message bus internal to EdgeX and has a specific message envelope that wraps all data published to this message bus.
+
+There currently are three implementations of the EdgeX MessageBus available to be used. These are `Redis Pub/Sub`(default), `MQTT` and `ZeroMQ`. The implementation type is selected via the `[Trigger.EdgexMessageBus]` configuration described below.
+
+### Type Configuration 
+
+
+!!! edgey "Edgex 2.0"
+    For EdgeX 2.0 the `SubscribeTopic` has been renamed to `SubscribeTopics` and moved under the **EdgexMessageBus** `SubscribeHost` section. The `PublishTopic` has also been moved under the **EdgexMessageBus** `PublishHost` section. Also the legacy `type` of `messagebus` has been removed.
```

Comment:
```suggestion
    For EdgeX 2.0 the `SubscribeTopic` has been renamed to `SubscribeTopics` and moved under the **EdgexMessageBus** `SubscribeHost` section. The `PublishTopic` has also been moved under the **EdgexMessageBus** `PublishHost` section.  So has been the legacy `type` of `messagebus`.
```

Reply:
No, that doesn't make sense, This have been renamed/moved.  `messagebus` have been removed.

Reply:
ok.